.evergreen/utils.sh

```diff
@@ -0,0 +1,36 @@
+# Usage:
+# venvcreate /path/to/python /output/path/for/venv
+# * param1: Python binary to use for the virtualenv
+# * param2: Path to the virtualenv to create
+venvcreate () {
```

Comment:
Note I copied this from pymongo (with some minor changes): https://github.com/mongodb/mongo-python-driver/blob/4.1.0/.evergreen/utils.sh#L5-L33